modules/favorites/endpoints/index.php

```diff
@@ -0,0 +1,98 @@
+<?php
+namespace Elementor\Modules\Favorites\Endpoints;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+use Elementor\Modules\Favorites\Module;
+use Elementor\Data\Base\Endpoint;
+use Elementor\Plugin;
+
+class Index extends Endpoint {
+
+	/**
+	 * @inheritDoc
+	 */
```

Comment:
```suggestion
```
We are not doing this here... it is useless, the IDE does his job. We should type consistent code.